addons/binding/org.openhab.binding.nest/README.md

```diff
@@ -98,31 +98,31 @@ The account Thing Type does not have any channels.
-| time_to_target_mins         | Number    | Time left to the target temperature (mins) approximately                                    |      R     |
-| set_point                   | Number    | The set point in degrees Celsius                                                            |     R/W    |
-| sunlight_correction_active  | Switch    | If sunlight correction is active                                                            |      R     |
-| sunlight_correction_enabled | Switch    | If sunlight correction is enabled                                                           |      R     |
-| using_emergency_heat        | Switch    | If the system is currently using emergency heat                                             |      R     |
+| Channel Type ID             | Item Type          | Description                                                                                 | Read Write |
+|-----------------------------|--------------------|---------------------------------------------------------------------------------------------|:----------:|
+| can_cool                    | Switch             | If the thermostat can actually turn on cooling                                              |      R     |
+| can_heat                    | Switch             | If the thermostat can actually turn on heating                                              |      R     |
+| fan_timer_active            | Switch             | If the fan timer is engaged                                                                 |     R/W    |
+| fan_timer_duration          | Number             | Length of time (in minutes) that the fan is set to run (15, 30, 45, 60, 120, 240, 480, 960) |     R/W    |
+| fan_timer_timeout           | DateTime           | Timestamp when the fan stops running                                                        |      R     |
+| has_fan                     | Switch             | If the thermostat can control the fan                                                       |      R     |
+| has_leaf                    | Switch             | If the thermostat is currently in a leaf mode                                               |      R     |
+| humidity                    | Number             | Indicates the current relative humidity                                                     |      R     |
+| last_connection             | DateTime           | Timestamp of the last successful interaction with Nest                                      |      R     |
+| locked                      | Switch             | If the thermostat has the temperature locked to only be within a set range                  |      R     |
+| locked_max_set_point        | Number:Temperature | The locked range max set point temperature                                                  |     R/W    |
```

Comment:
Is this change backwards compatible if not please document it for the upcoming 2.3 release

Reply:
Items without the Temperature unit still get updated but you can't use the automatic `%unit%` resolution or conversion when defining other units. 

So it's backwards compatible. :sun_with_face: